_source/templates/booky/config.scss

```diff
@@ -8,9 +8,10 @@ $color--dark: #3e3e3e;
 
 $color__background--primary: $color--primary;
 $color__background--dark: #2d2c2c;
-$color__background--medium: #3e3e3e;
+$color__background--medium: $color--dark;
 $color__background--ligth: #f5f5f5;
 $color__background--white: #fff;
```

Comment:
AHA! There it iiiiis, so you are also using a variable for basic colors, sooooo u can make use of this powerful weapon in your code as well! 🗡

Reply:
🛡